src/Microsoft.Extensions.Logging.Console/ConsoleLoggerFactoryExtensions.cs

```diff
@@ -12,68 +13,89 @@ public static class ConsoleLoggerExtensions
         /// <summary>
         /// Adds a console logger named 'Console' to the factory.
         /// </summary>
-        /// <param name="factory">The <see cref="LoggerFactory"/> to use.</param>
-        public static LoggerFactory AddConsole(this LoggerFactory factory)
+        /// <param name="builder">The <see cref="LoggerFactory"/> to use.</param>
+        public static ILoggerBuilder AddConsole(this ILoggerBuilder builder)
         {
-            factory.AddProvider("Console", new ConsoleLoggerProvider(factory.Configuration));
-            return factory;
+            builder.Services.AddSingleton<ILoggerProvider, ConsoleLoggerProvider>();
+
+            return builder;
+        }
+
+        /// <summary>
+        /// Adds a console logger named 'Console' to the factory.
+        /// </summary>
+        /// <param name="builder">The <see cref="LoggerFactory"/> to use.</param>
```

Comment:
`The <see cref="LoggerFactory"/> to use.` is no longer a good comment here, is it? (applies to all of the ILoggerBuilder based methods)